libraries/src/Application/AdministratorApplication.php

```diff
@@ -245,7 +245,7 @@ public function getTemplate($params = false)
 		{
 			$this->enqueueMessage(Text::_('JERROR_ALERTNOTEMPLATE'), 'error');
 			$template->params = new Registry;
-			$template->template = 'spring';
+			$template->template = 'khonsu';
```

Comment:
This should be adding khonsu not replacing atum - not sure how to do the code - see `libraries\src\Application\SiteApplication.php`

Reply:
The scripts were written only for `Atum` template, not for multiple templates support. I think it's an another issue which you can raise.

Reply:
Whatever way you look at it this code is wrong and should not be committed

Reply:
It was suggested to deal with the dual/multiple template issue at a later date